base/src/com/google/idea/blaze/base/run/smrunner/BlazeXmlToTestEventsConverter.java

```diff
@@ -351,7 +351,7 @@ private static void processTestCase(
       processor.onTestOutput(new TestOutputEvent(displayName, test.sysOut, true));
     }
     if (test.sysErr != null) {
-      processor.onTestOutput(new TestOutputEvent(displayName, test.sysErr, true));
+      processor.onTestOutput(new TestOutputEvent(displayName, test.sysErr, false));
```

Comment:
I'm not sure why we need to change this to `false`

Reply:
Last argument to `TestOutputEvent` here is `true` for outputting to stdOut, and `false` for stdErr.   
So we were reporting both sysOut and sysErr elements as standard output but it seems logical to differentiate between those and let Intellij present them differently?